src/snippets.ts

```diff
@@ -0,0 +1,83 @@
+import {ElementSnippet} from './editor-service';
+
+const snippets: ElementSnippet[] = [
+  {
+    'tagname': 'html-import',
+    'description': 'Template definition of an HTML import.',
+    'prefix': 'hi',
+    'expandTo': '<link rel="import" href="bower_components/.html">',
+    'expandToSnippet': '<link rel="import" href="${1:bower_components}/${2}/${2}.html">'
```

Comment:
Yes agreed, shall I extract an issue for that in the CLI? I think adding a line to the `bower.json` stating the type is the cleanest solution?